snap/snapcraft.yaml

```diff
@@ -42,3 +48,5 @@ parts:
 apps:
   amber-bash:
     command: bin/amber
+    environment:
+      STD_PATH: $SNAP/std
```

Comment:
This line looks good, but as @Ph0enixKM we should have a better approach, keep some default values in mind, because not all package manager can set a value in the environment I guess.